packages/devtools_app/lib/src/debugger/debugger_model.dart

```diff
@@ -784,14 +780,8 @@ class Variable extends TreeNode<Variable> {
 
     final value = this.value;
     if (value is InstanceRef) {
-      if (value.kind == InstanceKind.kList) {
-        return value.length ?? 0;
-      } else if (value.kind == InstanceKind.kMap) {
-        return value.length ?? 0;
-      } else if (value.kind != null && value.kind.endsWith('List')) {
-        // Catch-all for Uint8ClampedList, Uint8List, Uint16List, Uint32List,
-        // Uint64List, Int8List, Int16List, Int32List, Int64List, Flooat32List,
-        // Float64List, Inst32x3List, Float32x4List, and Float64x2List types:
+      if (value.kind != null &&
+          (value.kind.endsWith('List') || value.kind == InstanceKind.kMap)) {
```

Comment:
do we still need a check for `value.kind == InstanceKind.kList` here?

Reply:
Good point, done!